src/main/resources/templates/template_hmpps-restricted-patients-api.mustache

```diff
@@ -0,0 +1,24 @@
+<h1>Restricted Patients</h1>
+{{#.}}
+    <br>
```

Comment:
I've moved the `<br>` into here since otherwise there too much space for the _No data held_ scenario